Kickstarter-iOS/Features/ConfirmDetails/Controllers/PostCampaignPledgeSummaryViewController.swift

```diff
@@ -0,0 +1,171 @@
+    UIStackView(frame: .zero)
+      |> \.translatesAutoresizingMaskIntoConstraints .~ false
+  }()
+
+  private lazy var titleLabel: UILabel = { UILabel(frame: .zero) }()
+  private lazy var totalConversionLabel: UILabel = { UILabel(frame: .zero) }()
+  private lazy var totalStackView: UIStackView = { UIStackView(frame: .zero) }()
+  private let viewModel: PledgeSummaryViewModelType = PledgeSummaryViewModel()
+
+  // MARK: - Lifecycle
+
+  override func viewDidLoad() {
+    super.viewDidLoad()
+
+    self.configureSubviews()
+
+    self.viewModel.inputs.viewDidLoad()
+  }
+
+  // MARK: - Styles
+
+  override func bindStyles() {
+    super.bindStyles()
+
+    _ = self.view
+      |> checkoutBackgroundStyle
+
+    _ = self.rootStackView
+      |> rootStackViewStyle
+
+    _ = self.titleAndTotalStackView
+      |> adaptableStackViewStyle(
+        self.traitCollection.preferredContentSizeCategory.isAccessibilityCategory
+      )
+      |> titleAndTotalStackViewStyle
+
+    _ = self.totalStackView
+      |> totalStackViewStyle(self.traitCollection.preferredContentSizeCategory.isAccessibilityCategory)
+
+    _ = self.titleLabel
+      |> titleLabelStyle
+
+    _ = self.amountLabel
+      |> amountLabelStyle
+
+    _ = self.totalConversionLabel
+      |> totalConversionLabelStyle
+  }
+
+  private func configureSubviews() {
+    _ = (self.rootStackView, self.view)
+      |> ksr_addSubviewToParent()
+      |> ksr_constrainViewToEdgesInParent()
+
+    _ = ([self.titleAndTotalStackView], self.rootStackView)
+      |> ksr_addArrangedSubviewsToStackView()
+
+    _ = ([self.titleLabel, self.totalStackView], self.titleAndTotalStackView)
+      |> ksr_addArrangedSubviewsToStackView()
+
+    _ = ([self.amountLabel, self.totalConversionLabel], self.totalStackView)
+      |> ksr_addArrangedSubviewsToStackView()
+  }
+
+  // MARK: - View model
+
+  internal override func bindViewModel() {
+    super.bindViewModel()
+
+    self.viewModel.outputs.notifyDelegateOpenHelpType
+      .observeForControllerAction()
+      .observeValues { [weak self] helpType in
+        self?.presentHelpWebViewController(with: helpType, presentationStyle: .formSheet)
+      }
+
+    self.amountLabel.rac.attributedText = self.viewModel.outputs.amountLabelAttributedText
+    self.totalConversionLabel.rac.text = self.viewModel.outputs.totalConversionLabelText
+  }
+
+  // MARK: - Configuration
+
+  internal func configure(with data: PledgeSummaryViewData) {
+    self.viewModel.inputs.configure(with: data)
+  }
+}
+
+extension PostCampaignPledgeSummaryViewController: UITextViewDelegate {
+  func textView(
+    _: UITextView, shouldInteractWith _: NSTextAttachment,
+    in _: NSRange, interaction _: UITextItemInteraction
+  ) -> Bool {
+    return false
+  }
+
+  func textView(
+    _: UITextView, shouldInteractWith url: URL, in _: NSRange,
+    interaction _: UITextItemInteraction
+  ) -> Bool {
+    self.viewModel.inputs.tapped(url)
+    return false
+  }
+}
+
+// MARK: - Styles
```

Comment:
Nit: I find styles to be more readable and they compile faster when we don't use prelude to define them. Thoughts on using standard UI kit (ex using helper functions) to style them instead?